src/lib/scripts/voting-gauge.generator.ts

```diff
@@ -367,27 +414,34 @@ async function getGaugeAddress(
 (async () => {
   console.log('Generating voting-gauges.json...');
 
-  let votingGauges = await Promise.all(
-    POOLS.map(async ({ id, network }) => {
-      const address = await getGaugeAddress(id, network);
-      const pool = await getPoolInfo(id, network);
-
-      const tokenLogoURIs = {};
-      for (let i = 0; i < pool.tokens.length; i++) {
-        tokenLogoURIs[pool.tokens[i].address] = await getTokenLogoURI(
-          pool.tokens[i].address,
-          network
-        );
-      }
+  let votingGauges: VotingGauge[] = [];
 
-      return {
-        address,
-        network,
-        pool,
-        tokenLogoURIs,
-      };
-    })
-  );
+  for (const { id, network } of POOLS) {
+    const gauge = await getGaugeInfo(id, network);
+    if (!gauge) continue;
+
+    if (gauge.isKilled) {
+      const weight = await getGaugeRelativeWeight(gauge.address);
```

Comment:
We're not really taking advantage of the multicaller here. Maybe it's not important? But to save some requests, if you could get all the gauge addresses in an array you could do something like:
```ts
const weights = await getGaugeRelativeWeights(gaugeAddresses);
// Then run the pools loop
for (const { id, network } of POOLS) {
    ...
    if (gauge.isKilled) {
        const weight = weight[gauge.address];
         ...
```
But I understand it might not be feasible or efficient to fetch all the gauge addresses first.

You would also have to update the use of the multicaller in `getGaugeRelativeWeight` to make this work too.